app/views/content_change_requests/_request_details.html.erb

```diff
@@ -20,6 +20,33 @@
   </span>
 </div>
 
+<div class="form-group">
+  <span class="form-label">
+    <%= f.label :reason, "What's the reason for the request?" %>
+  </span>
+  <p class="help-block hint-block--md-6" id="reason-hint">
+    If none of the options below are appropriate, please select other.
+  </p>
+  <span class="form-wrapper">
+    <%= f.select :reason,
```

Comment:
Do we not need to update [this test](https://github.com/alphagov/support/blob/7e62c6694cda6bf719284e51aaf2fcba131e09d9/spec/features/content_change_requests_spec.rb#L73-L83) to select a reason?